### PR TITLE
Fix android http client memory leak

### PR DESCRIPTION
**Pull request by hueypark**

### CODE REVIEW

**Comment by minggo** [comment]
@samuele3hu 
Please take a look.

**Comment by samuele3hu** [comment]
[ci rebuild]

File: cocos/network/HttpClient-android.cpp [hunk 1]
```diff
@@ -422,6 +422,7 @@ class HttpURLConnection
             jobject jObj = methodInfo.env->CallStaticObjectMethod(methodInfo.classID, methodInfo.methodID, jurl);
             _httpURLConnection = methodInfo.env->NewGlobalRef(jObj);
             methodInfo.env->DeleteLocalRef(jurl);
+            methodInfo.env->DeleteLocalRef(jObj);
```
Comment (samuele3hu) [inline cocos/network/HttpClient-android.cpp:425]: Why use DeleteLocalRef to delete a GlobalRef?

**Comment by hueypark** [comment]
I think _httpURLConnection is GlobalRef and jObj is LocalRef.
And my project was shut down when call http client. (512 times)

**Comment by WenhaiLin** [comment]
I agree with @HueyPark .

**Comment by samuele3hu** [comment]
Ok,I know.I think it is no problem.

**Pull request merged**